src/app/components/sectioned-form/index.html

```diff
@@ -19,22 +19,37 @@
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Sectioned form section properties">
+    <sky-demo-page-property
+      propertyName="heading"
+    >
+    Specifies the section header. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="active"
       isOptional="true"
+      defaultValue="false"
     >
-    Indicates whether the section is active when the form loads.
+    Indicates whether the section is active when the form loads. This property accepts <stache-code>boolean</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="heading"
+      propertyName="ariaInvalid"
```

Comment:
I just noticed that while `ariaInvalid` and `ariaRequired` are properties of the vertical tabs, they aren't called the same thing in sectioned form sections. They're actually called `fieldInvalid` and `fieldRequired `. Who knows why?

Anyways, we'll probably need to update this to the latter above. I think your descriptions still hold true, because they just pass along the values to the tab's aria inputs. See here: https://github.com/blackbaud/skyux-tabs/blob/master/src/app/public/modules/sectioned-form/sectioned-form-section.component.html#L4

Reply:
Ah, thanks for the heads up! Fixing that now